test/relaxed-simd.wast

```diff
@@ -0,0 +1,115 @@
+(module
```

Comment:
I think @tlively is trying to get new tests added to `lit/`. See for example `test/lit/table-operations.wast` which does binary and text format stuff.

Reply:
Done, copied paste the lit stuff and ran script to update the assertions.